linux_sql/scripts/psql_docker.sh

```diff
@@ -0,0 +1,47 @@
+#! /bin/sh
+
+cmd=$1
+db_username=$2
+db_password=$3
+
+sudo systemctl status docker || systemctl start docker
+#docker container inspect jrvs-psql
+
+docker ps -f name=jrvs-psql
+container_status=$?
+echo $?
+
+case $cmd in
+  create)
+
+  if [ $container_status -eq 0 ]; then
+		echo 'Container already exists'
+		exit 1
```

Comment:
this can exit 0. Since the container already exists it doesnt need to be created; nothing has really gone wrong and we dont expect any special action from the user.